_po/ja/how-to/laravel.po

```diff
@@ -574,6 +865,13 @@ msgid ""
 "                                      </dt>\n"
 "                                    </div>"
 msgstr ""
+"                                    <div class=\"border-t border-gray-200 sm:co"
+"l-span-4\">\n"
+"                                      <dt class=\"text-sm sm:text-base font-med"
+"ium text-gray-900\">\n"
+"                                        {{ __('Body') }}\n"
+"                                      </dt>\n"
+"                                    </div>"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.